bot/src/player/item.rs

```diff
@@ -32,4 +32,18 @@ impl Item {
             },
         }
     }
+
+    pub fn spotify_playable(&self) -> bool {
```

Comment:
Nit, but: Simply `is_playable` is sufficient, it just happens to be true for all YouTube videos :).